cirq-google/cirq_google/transformers/analytical_decompositions/two_qubit_to_sycamore_test.py

```diff
@@ -86,7 +86,10 @@ def test_known_two_qubit_op_decomposition(op, theta_range):
         cirq.FSimGate(0.25, 0.85).on(*_QUBITS),
         cirq.XX(*_QUBITS),
         cirq.YY(*_QUBITS),
-        *[cirq.testing.random_unitary(4, random_state=1234) for _ in range(10)],
+        *[
+            cirq.MatrixGate(cirq.testing.random_unitary(4, random_state=1234)).on(*_QUBITS)
+            for _ in range(10)
+        ],
```

Comment:
This repeats the same test 10 times, because random_unitary is the same for constant random_state.  Please replace with
```suggestion
        cirq.MatrixGate(cirq.testing.random_unitary(4)).on(*_QUBITS),
```

(The random_unitary would be still reproducible via the `--randomly-seed` pytest option)